### PR TITLE
Adding release workflow 

### PR DESCRIPTION
**Pull request by galleon**
The workflow is triggered on v tags

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+      - name: generate documentation
+        run: yarn global add vuepress && yarn install && yarn docs:build && touch docs/.vuepress/dist/.nojekyll
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          ACCESS_TOKEN: ${{ secrets.gh_access_token }}
+          BRANCH: gh-pages # The branch the action should deploy to.
+          FOLDER: docs/.vuepress/dist # The folder the action should deploy.
+          GIT_CONFIG_EMAIL: guillaume.alleon@gmail.com
```
Comment (fteicht) [inline .github/workflows/release.yml:317]: @galleon To change to an Airbus email address as soon as we have one.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Publish distribution 📦 to PyPI
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_PASSWORD }}
```
Comment (fteicht) [inline .github/workflows/release.yml:264]: @galleon Which account does it use by default (say when the script is run from Airbus' scikit-decide repository)?

File: .github/workflows/release.yml (hunk 1 above)
Comment (galleon) [reply .github/workflows/release.yml:317]: Yes, you can always open an issue as a reminder.

File: .github/workflows/release.yml (hunk 2 above)
Comment (galleon) [reply .github/workflows/release.yml:264]: It uses `scikit-decide` project on PyPI. The secret can be changed from github admin page. For the time being we are both users on this `scikit-decide` project.

**Review by fteicht — APPROVED** [review]
It's good for me, thanks for your work @galleon !

**Pull request merged**